Packages/MIES/MIES_WaveDataFolderGetters.ipf

```diff
@@ -6929,15 +6929,15 @@ Function/WAVE GetTPSettingsLabnotebookKeyWave(string device)
 	SetDimLabel 0, 2, Tolerance, wv
 
 	wv[%Parameter][0]  = "TP Baseline Fraction" // fraction of total TP duration
-	wv[%Units][0]      = ""
+	wv[%Units][0]      = "%"
```

Comment:
We store a number between 0.25 and 0.49 and for me these are not `%`. We don't just store what the DAEphys GUI says.